privacy/anonymizing-your-mac-address.md

```diff
@@ -170,3 +172,48 @@ To disable MAC Randomizing if you find that a network connecting to does not lik
 - Also in `fedora-23` type `sudo systemctl disable macspoof@enp0s0`
 - Remove the service file `sudo rm /etc/systemd/system/macspoof@.service` in TemplateVM
 - Delete the package `sudo dnf remove macchanger`
+
+Method 2
+========
+
+An alternative approach is to use udev rules.
+
+This method does not require you to specify interface names, will apply to all interfaces, even those added after boot, and will retain the random MAC address through sleep/wake cycles.
+
+Install **macchanger** package in the TemplateVM backing sys-net.
+
+In that template create a udev rule using an editor of your choice.
+
+In /etc/udev/rules.d create the file 00-mac-spoof.rules with content:
+```
+SUBSYSTEM=="net", ACTION=="add", ATTR{type}=="1", RUN+="/usr/bin/macchanger -e  $name"
```

Comment:
@marmarek Thanks for that - I don't generally use Network Manager, so I haven't seen any effects like this. That said, I have it on one machine with NM and that just works fine - that is, it doesn't appear as if NM is running a DHCP client on the downlink. Let me dust that off and have a look.